libpyvinyl/BaseCalculator.py

```diff
@@ -56,16 +60,16 @@ class BaseCalculator(AbstractBaseClass):
     backengine as all other ViNYL Calculators.
 
     A complete example including a instrument and calculators can be found in
-    `test/integration/plusminus`
+    ``test/integration/plusminus``
 
     """
 
     def __init__(
         self,
         name: str,
-        input: Union[DataCollection, list, BaseData],
+        input: Union[DataCollection, List[DataCollection], BaseData],
```

Comment:
input cannot be a list of `DataCollection` while the return of the iterator has to be a `BaseData`.
```suggestion
        input: Union[DataCollection, List[DataCollection], BaseData],
```

Reply:
Understood